modules/student.rb

```diff
@@ -1,11 +1,18 @@
 require_relative 'person'
 
 class Student < Person
+  attr_reader :classroom
+
   def initialize(classroom, age, name = 'unknown', parent_permission: true)
     super(age, name, parent_permission)
     @classroom = classroom
   end
 
+  def classroom=(classroom)
+    @classroom = classroom
+    classroom.students.push(self) unless classroom.students.includes?(self)
```

Comment:
- [ ] Please 🙏🏽 correct the method `include` instead of `includes`.

Reply:
Thanks for the feedback. All the requested changes have been implemented.